pkg/kritis/util/allowlist.go

```diff
@@ -39,20 +41,104 @@ func RemoveGloballyAllowedImages(images []string) []string {
 	return notAllowlisted
 }
 
-func imageInAllowlist(image string) (bool, error) {
-	for _, w := range constants.GlobalImageAllowlist {
-		allowlistRef, err := name.ParseReference(w, name.WeakValidation)
+// RemoveGloballyAllowedImages returns all images that aren't in gap allowlists
+func RemoveGapAllowedImages(images []string, allowlist []string) []string {
```

Comment:
Same question: does this file contain anything I should pay special attention to, give that I reviewed the allowlist change in your branch?

Reply:
No. All changes are are new in this PR so far are due to merge fixes, IIRC.